apmpackage/apm/changelog.yml

```diff
@@ -1,4 +1,14 @@
-- version: generated
+- version: "generated"
```

Comment:
**question**: why was this changed ? Mostly asking because the 8.6 PR is using `generated` (https://github.com/elastic/apm-server/pull/9576/files)

Reply:
but on 8.6 branch it is also `"generated"` and i don't think it makes a difference in yml anyway